packages/app/app/containers/IpcContainer/index.js

```diff
@@ -98,8 +98,9 @@ class IpcContainer extends React.Component {
 
       try {
         const { artist, name, thumbnail } = this.props.queue.queueItems[this.props.queue.currentSong];
+        const duration = this.props.queue.queueItems[this.props.queue.currentSong].streams?.[0]?.duration;
```

Comment:
Adding `streams` to the destructor above and then making it `streams?.[0]?.duration` would look nicer.